### DISCUSSION TITLE
media player entity attributes too late update

### DISCUSSION THREAD

**Discussion by maverik80** [alandtse/alexa_media_player#1442] [Q&A]
good evening, I was trying to make a template with my two echo shows and as a result in a button card I wanted the cover of the song played in the background, but the media player entities really send the attributes and also its status with a monstrous delay even if already in playing results in paused / standby for about 10 minutes. solutions?

**Comment by ian-tait** [comment]
It would seem we have similar issues. At least your entities updates after a while. 
Please check out #1544 and let me know if you think this is the same thing?

I feel this may relate to how you sign in to the Alexa service. 
Do you use the built-in 2FA or the OTP (mobile authenticator)?
I know the most common mistake is related to the Amazon service location, so I have made sure I use .co.uk; which one have you signed up to?
